MultiplayerTestRunner/include/ErrorCodes.h

```diff
@@ -23,8 +23,8 @@
 namespace MultiplayerTestRunner::ErrorCodes
 {
 constexpr const int SUCCESS = 0;
-constexpr const int COULD_NOT_FIND_CREDENTIALS_FILE = 100;
-constexpr const int MALFORMED_CREDENTIALS_FILE = 101;
+constexpr const int FAILED_TO_CREATE_USER = 100;
+constexpr const int UNUSED_ERROR_CODE = 101;
```

Comment:
[nit] I'd leave it as the old name, but comment it out 
```suggestion
// constexpr const int MALFORMED_CREDENTIALS_FILE = 101; --Credentials file no longer used
```

Reply:
I think I ended up using it, so not actioning for now. will do if it turns out I don't need it